api/v1beta1/azuremanagedcluster_webhook.go

```diff
@@ -57,24 +52,6 @@ func (r *AzureManagedCluster) ValidateCreate() (admission.Warnings, error) {
 
 // ValidateUpdate implements webhook.Validator so a webhook will be registered for the type.
 func (r *AzureManagedCluster) ValidateUpdate(oldRaw runtime.Object) (admission.Warnings, error) {
-	old := oldRaw.(*AzureManagedCluster)
-	var allErrs field.ErrorList
-
-	// custom headers are immutable
```

Comment:
ASO does not support adding custom headers to requests, so all of that is now gone.

Reply:
That is a bummer, we discussed this on the community call as well.  We will need to update [our documentation](https://capz.sigs.k8s.io/topics/managedcluster#enable-aks-features-with-custom-headers---aks-custom-headers) and put a release note to make sure people know it is no longer supported.

Reply:
Good call, I'll update the docs.

@CecileRobertMichon Do you know if there's a decent way to format release notes longer than one line? Or perhaps add multiple release notes for a single PR?

Reply:
Should we add a comment here as a TODO to add it back when it is supported?

Reply:
It sounds like there's a good chance ASO will never support custom headers.